error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/MapRules.java

```diff
@@ -43,6 +45,23 @@ V after(Map<K, V> map, T key) {
     }
   }
 
+  /**
+   * Prefer {@link Map#getOrDefault(Object, Object)} over more contrived alternatives. Note that
+   * this method may return null instead of throwing a NPE.
```

Comment:
Good point! (I'll move this to an `XXX` comment so that we can find it back more easily later; we have some plans around calling this sort of thing out more prominently.)

Reply:
Hmm, I'm not completely agreeing with this though. Sometimes we do this in a separate `<p>` sentence in the Javadoc. Will leave it for now though, apart from a minor fix :).